app/assets/javascripts/hydra-editor/editMetadata.js

```diff
@@ -1,3 +1,3 @@
 Blacklight.onLoad(function() {
-  $('.multi_value.form-group').manage_fields();
+  $('.repeating-field.form-group').manage_fields();
```

Comment:
In the wrapper html, I actually add a new class.  `.multi_value` is simple_form magic based on the input type's name, and it'll be done the same as it already is today.  The trick is that if you define a new input type, but want the same logic (which we're very likely to need), the custom input type has to lie to simple form about its type name in order to get that class in the CSS

So if there's code in the wild that relies on .multi_value for custom styling, it'll be fine - that class is still there since it's simple_form magic.